project.py

```diff
@@ -0,0 +1,174 @@
+    inputs: a two key-word dictionary, a one key-word dictionary, and a string of all scripts
+    outputs: a string that is the "final" speech that Trump would make"""
+    spech = random.randrange(300, 500) #generates a random speech of length randomly selected in this range (in words)
+    speech = []
+    listowords = allwordss.split()
+    speech.append(random.choice(listowords).lower())
+    speech.append(random.choice(yup2[speech[0]]))
+    speech[0] = speech[0].capitalize()
+    for i in range(2, spech-1):     #Make a speech of length spech through referencing the dictionaries yup and yup2
+        checker = (speech[i-2].lower(), speech[i-1].lower())
+        if '.' in (speech[i-1]) or ('?' in (speech[i-1])) or ('!' in (speech[i-1])):
+            speech.append(random.choice(yup[checker]).capitalize())
+        else:
+            appendix = random.choice(yup[checker])
+            if appendix in "i'm i i'll i've":   #capitalize I
+                appendix = appendix.capitalize()
+            speech.append(appendix)
+    index = 0
+    for x in speech:        #A variety of statements that change Family Guy specific phrases to those that Trump could use
+        xl = x.lower()
+        if "peter" in xl:
+            speech[index] = "Trump"
+        elif "lois" in xl:
+            speech[index] = "Ivanka"
+        elif "brian" in xl:
+            speech[index] = "Pence"
+        elif "stewie" in xl:
+            speech[index] = "Obama"
+        elif "quagmire" in xl:
+            speech[index] = "ISIS"
+        elif "meg" in xl:
+            speech[index] = "Mexico"
+        elif "chris" in xl:
+            speech[index] = "China"
+        elif "quahog" in xl:
+            speech[index] = "MURICA"        #This is more likely to sound like Trump than "America"
+        elif "griffin" in xl:
+            speech[index] = "President"
+        if '.' in x:
+            temp = x
+            mini = temp.split('.')
+            speech[index] = mini[0]
+            speech.insert(index, mini[1].capitalize())
+        index += 1
+    index = 0
+    for y in speech:    #Fixes some of the spacing issues
+        if " " in y:
+            speech[index] = y.replace(' ', '')
+        index += 1
+    if ('.' in speech[-1]) or ('!' in speech[-1]) or ('?' in speech[-1]):   #how to determine if punctuation needs to be added at the end
+        product = ' '.join(speech)
+    else:
+        product = ' '.join(speech) + '.'
+    product = product.replace('  ', ' ')        #replace all double spaces with single spaces.
+    return product
+
+if __name__ == '__main__':
+    max_season = 3
+    allscripts = ''
+    my_file = Path("dict1.p")
+    my_file2 = Path("dict2.p")
+    my_file3 = Path("allwords.p")
+    remake = False
+    y = input("Do you want to customize the number of seasons? (y/n)")
+    if y.lower() == 'y':
+        max_seasons = input("How many seasons do you want?: ")
+        remake = True
+    # print(not my_file.is_file()) #For testing purposes
```

Comment:
For cleanliness/best practices, removing commented-out code and print-statement-debugging artifacts would be a good idea.